### PR TITLE
Minor visual changes

### PR DESCRIPTION
**Pull request by boyankostov**
Note: Reference image below

## Intro spacing changes
Looking at (1):
- I've fixed the margins
- Made stylistic changes to the button, adding its SVG icon
- The font is now bold

## Webkit scrollbar modifications
- Minified the scrollbar, it now shows only on :hover. (webkit only)

## Header layout change
- flexbox > css grid to accommodate the new design

screenshot 2017-12-19 12 39 08

### CODE REVIEW

File: src/app/header/header.component.ts [hunk 1]
```diff
@@ -17,26 +25,42 @@ import { StoreService } from '../store.service';
     <a *ngIf="(store.user$ | async) != null" routerLink="/" (click)="logout()">Изход</a>
     -->
 
+    <!--
     <div class="toggle border-primary br-md pointer" (click)="showSidebarChange.emit(!showSidebar)">
       <img class="toggle__image" [src]="showSidebar ? 'assets/chevron-up.svg' : 'assets/chevron-down.svg'" />
     </div>
+    -->
```
Comment (vdsabev) [inline src/app/header/header.component.ts:32]: Have you considered how the mobile design would work if there's no toggle button?

File: src/app/header/header.component.ts [hunk 2]
```diff
@@ -6,6 +6,14 @@ import { StoreService } from '../store.service';
 @Component({
   selector: 'Header',
   template: `
+    
+    <!-- Този back btn се появява само във формата, но това е отвъд моите способности :) -->
```
Comment (vdsabev) [inline src/app/header/header.component.ts:10]: We're open to pull requests from all nationalities, let's use English everywhere except for the i18n object definitions :)

File: src/app/sidebar/sidebar.component.ts [hunk 3]
```diff
@@ -39,6 +40,15 @@ import { Subscription } from 'rxjs/Subscription';
         height: calc(100vh - var(--header-height) - 20px);
       }
     }
+
+    .content::-webkit-scrollbar {
```
Comment (vdsabev) [inline src/app/sidebar/sidebar.component.ts]: When is removing the scrollbar beneficial?

File: src/app/header/header.component.ts (hunk 1 above)
Comment (boyankostov) [reply src/app/header/header.component.ts:32]: Yep, I'll try to work it out like this:

image

File: src/app/sidebar/sidebar.component.ts (hunk 3 above)
Comment (boyankostov) [reply src/app/sidebar/sidebar.component.ts]: Actually only on desktop. Even if it's not removed, it's good to have it visually less prominent.

File: src/app/header/header.component.ts (hunk 1 above)
Comment (vdsabev) [reply src/app/header/header.component.ts:32]: This looks great! 😄

File: src/app/app.component.ts [hunk 4]
```diff
@@ -43,7 +43,7 @@ import { Component } from '@angular/core';
       .sidebar {
         top: 10px;
         left: 10px;
-        height: auto;
+        height: calc(100% - 20px);
```
Comment (vdsabev) [inline src/app/app.component.ts]: Perhaps you wouldn't need to set height here if you set bottom to 0?

**Review by vdsabev — APPROVED** [review]

**Pull request merged**